File: QuanLiNhaHang_C#/QuanLyNhaHang/KhachHang.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyNhaHang
{
    public class KhachHang
    {
        // Automatic properties
        public string maKhachHang { get; set; }
        public string tenKH { get; set; }
        public string diaChi { get; set; }
        public string soDienThoai { get; set; }
        // Constructor
        public KhachHang() { }
        public KhachHang(KhachHang KH)
        {
            this.maKhachHang = KH.maKhachHang;
            this.tenKH = KH.tenKH;
            this.diaChi = KH.diaChi;
            this.soDienThoai = KH.soDienThoai;
        }
        public KhachHang(string maKhachHang, string tenKH, string diaChi, string soDienThoai)
        {
            this.maKhachHang = maKhachHang;
            this.tenKH = tenKH;
            this.diaChi = diaChi;
            this.soDienThoai = soDienThoai;
        }

    }
}

File: QuanLiNhaHang_Java/src/mypack/KhachHang.java
package mypack;

import java.util.ArrayList;
import java.util.List;

public class KhachHang 
{
	// Automatic Properties
	public String maKhachHang;
	public String tenKH;
	public String diaChi;
	public String soDienThoai;
	
	// Constructors
	public KhachHang() { }
	public KhachHang(KhachHang KH)
	{
		this.maKhachHang = KH.maKhachHang;
		this.tenKH = KH.tenKH;
		this.diaChi = KH.diaChi;
		this.soDienThoai = KH.soDienThoai;
	}
	public KhachHang(String maKhachHang, String tenKH, String diaChi,
			String soDienThoai)
	{
		this.maKhachHang = maKhachHang;
		this.tenKH = tenKH;
		this.diaChi = diaChi;
		this.soDienThoai = soDienThoai;
	}
	
	
	// getter -setter
	public String getmaKhachHang()
	{
		return maKhachHang;
	}
	public void setmaKhachHang(String maKhachHang)
	{
		this.maKhachHang = maKhachHang;
	}
	public String getTenKH()
	{
		return tenKH;
	}
	public void setTenKH(String tenKH)
	{
		this.tenKH = tenKH;
	}
	public String getDiaChi()
	{
		return diaChi;
	}
	public void setDiaChi(String diaChi)
	{
		this.diaChi = diaChi;
	}
	public String getSoDienThoai()
	{
		return soDienThoai;
	}
	public void setSoDienThoai(String soDienThoai)
	{
		this.soDienThoai = soDienThoai;
	}
}
